readthedocs/projects/apps.py

```diff
@@ -13,3 +17,13 @@ def ready(self):
         import readthedocs.projects.tasks.builds  # noqa
         import readthedocs.projects.tasks.search  # noqa
         import readthedocs.projects.tasks.utils  # noqa
+
+        # HACK: remove djstripe initial checks.
+        # It performs checks that require access to the database, which make the builders to fail.
+        # Note this code has to be executed _after_ djstripe has been initialized.
```

Comment:
```suggestion
        # Note this is placed in the `projects` app, since it's listed after `djstripe` in `INSTALLED_APPS`, as this code has to be executed _after_ djstripe has been initialized.
```